app/forms/personal_details_form.rb

```diff
@@ -0,0 +1,34 @@
+class PersonalDetailsForm
+  include ActiveModel::Model
+
+  attr_accessor :fullname, :email, :licence_id
+
+  def initialize(fullname: nil, email: nil, licence_id: nil)
+    @fullname = fullname
+    @email = email
+    @licence_id = licence_id
+  end
+
+  validates :fullname,
+    presence: {
+      message: "You must provide your full name"
```

Comment:
Some of these messages could be more aligned with the examples https://design-system.service.gov.uk/components/text-input/#if-the-input-is-empty

So this would be "Enter your full name"

Reply:
Thanks, you are right.